R/aggregate.R

```diff
@@ -114,50 +108,19 @@ agg_climate_data <- function(year, data_raster, climate_var, daily_agg, trans =
     clim_daily <- clim_daily - 273.15
```

Comment:
can we put the Kelvin conversion factor up at the top of a script somewhere and only call it from there? I am nervous about hardcoding this many times (it was already messed up once in a previous iteration)

Reply:
Anna and I were discussing the conversions and thought that maybe we don't want to do any of this, and that its on the user to change units if they desire.